tetris.py

```diff
@@ -172,7 +189,8 @@ def block_mover(self):
     def block_rotator(self, event):
         rotate = []
         for (x, y) in self.current_block:
-            rotate.append((y, x))
+            if self.current_block == 'I':
+                rotate.append((y, x))
```

Comment:
This deletes all blocks except `I` blocks. Don't try to fix this function in the same pull request.

Reply:
Actually, this doesn't change what the program does at all: `self.current_block == 'I'` is always false, because `self.current_block` is not a string.